basic/atm/ATM.py

```diff
@@ -0,0 +1,106 @@
+class ATM:
+    PATH_SAVING_FILE = "ATM_data_base.txt"
+
+    def __init__(self):
+        self._accounts_dict = {}
+        with open(ATM.PATH_SAVING_FILE, "r") as f:
+
```

Comment:
delete this empty line, and put it before the for line,
it will make be more readable